### ISSUE TITLE
Workflow Log does not get copied over to S3 after workflow finished

### ISSUE THREAD

**Issue by asmoe4** [broadinstitute/cromwell#4446]
I'm running a WDL workflow using Cromwell AWS in run mode.  I noticed that workflow level log that is generated is actually on the local disk where the workflow is running.  The final workflow log is not being copied over to S3.

Also, if I provide a s3 path in `workflow-log-dir`, it will just create a folder locally.  Is there a way to get the workflow log copied over to s3 automatically after the workflow is finished?

**Comment by ruchim** [comment]
Hey @asmoe4 

It would help to confirm that your Cromwell config contains a stanza that looks like:

```
engine {
  filesystems {
    s3 {
      auth = "default"
    }
  }
}
```

**Comment by asmoe4** [comment]
> If you are calling Cromwell in run mode, can you wrap it in a script followed by an AWS CLI command to copy the workflow log?

Yes -- I could do that.  I was just thinking that workflow log should be automatically copied over since this is a cromwell workflow level log?  But if it's too much to implement, the workaround will be just manually copying over it at the end of the workflow.

**Comment by asmoe4** [comment]
> Hey @asmoe4
> 
> It would help to confirm that your Cromwell config contains a stanza that looks like:
> 
> ```
> engine {
>   filesystems {
>     s3 {
>       auth = "default"
>     }
>   }
> }
> ```

@ruchim  -- Yes, I have the above configuration defined in my config file.
